File: solutions/0745-find-smallest-letter-greater-than-target/solution.js
var nextGreatestLetter = function(letters, target) {
    let targets=target.charCodeAt(0);
    let list=[];
    for(i=0;i<letters.length;i++){
        let value=letters[i].charCodeAt(0);
        if(value>targets){
            list.push(letters[i])
        }
    }
  if(list.length == 0){
    return letters[0];
  }
  else{
    return list.sort()[0];
  }

 }

File: solutions/0745-find-smallest-letter-greater-than-target/solution.py
class Solution:
    def nextGreatestLetter(self, letters: List[str], target: str) -> str:
        target_code = ord(target)
        greater_letters = []

        for letter in letters:
            if ord(letter) > target_code:
                greater_letters.append(letter)
  
        if len(greater_letters)==0:
            return letters[0]

        return min(greater_letters)

